lua/themepark/plugins/bbox.lua

```diff
@@ -0,0 +1,282 @@
+        if layer.tiles ~= false then
+            local layer_config = build_layer_config(layer)
+            if layer_config then
+                table.insert(layers, layer_config)
+            end
+        end
+    end
+    for _, data in ipairs(extra_layers) do
+        if not data.geometry_field then
+            data.geometry_field = 'geom'
+        end
+        if not data.srid then
+            data.srid = plugin.themepark.options.srid
+        end
+        table.insert(layers, data)
+    end
+    return layers
+end
+
+local function ordered_keys(o)
+    local keys1 = {}
+    local keys2 = {}
+    for k, v in pairs(o) do
+        if type(v) == 'table' then
+            table.insert(keys2, k)
+        else
+            table.insert(keys1, k)
+        end
+    end
+    table.sort(keys1)
+    table.sort(keys2)
+    for _,v in ipairs(keys2) do
+        table.insert(keys1, v)
+    end
+    return keys1
+end
+
+local function dump_toml(o, indent_size, parents)
+    indent_size = indent_size or 2
+    parents = parents or {}
+    if type(o) == 'table' then
+        local s = ''
+        local indent = string.rep(" ", #parents * indent_size)
+        -- sort keys for stable output order
+        for _, k in ipairs(ordered_keys(o)) do
+            local v = o[k]
+            if type(v) == 'table' then
+                local array_val = v[1] ~= nil
+                if array_val then
+                    table.insert(parents, k)
+                    s = s .. dump_toml(v, indent_size, parents)
+                    table.remove(parents)
+                elseif type(k) == 'number' then
+                    local tag = table.concat(parents, ".")
+                    s = s .. '\n\n' .. indent .. '[[' .. tag ..']]' .. dump_toml(v, indent_size, parents)
+                else
+                    table.insert(parents, k)
+                    indent = string.rep(" ", #parents * indent_size)
+                    local tag = table.concat(parents, ".")
+                    s = s .. '\n\n' .. indent .. '[' .. tag ..']' .. dump_toml(v, indent_size, parents)
+                    table.remove(parents)
+                end
+            else
+                if type(v) == 'string' then
+                    if string.find(v, '"') then
+                        v = '"""'..v..'"""'
+                    else
+                        v = '"'..v..'"'
+                    end
+                end
+                s = s .. '\n' .. indent .. k ..' = ' .. dump_toml(v, indent_size, parents)
+            end
+        end
+        return s
+    else
+        return tostring(o)
+    end
+end
+
+-- ---------------------------------------------------------------------------
+-- ---------------------------------------------------------------------------
+function plugin:write_config(filename, options)
+    if not options then
+        options = {}
+    end
+
+    local config = {
+        webserver = { server_addr = '0.0.0.0:8080' },
```

Comment:
Default should probably be 127.0.0.1 to not open up a server to the world.

This is probably something we want to make configurable. The t-rex plugin also needs those settings. But okay for now. We can do that after this has been merged.

Reply:
Changed to 127.0.0.1 (0.0.0.0 is needed for Docker setups).